### PR TITLE
refactor swipe offset

### LINKED ISSUE #20: Add api for changing swipe offset
Add function in DSL for changing the default offset settings on standard swipes

### PR DESCRIPTION
**Pull request by alexander-kulikovskii**
closes #20

### CODE REVIEW

File: lib/src/main/java/fi/epicbot/toster/executor/android/AndroidExecutor.kt [hunk 1]
```diff
@@ -105,7 +105,11 @@ internal open class AndroidExecutor(
             // TODO make measurement only once
             val rawData = "wm size".adbShell()
             val rawSize = rawData.replace("Physical size: ", "").split("x")
-            swipeMove.toMove(width = rawSize[0].toInt(), height = rawSize[1].toInt())
+            swipeMove.toMove(
+                swipeOffset = config.swipeOffset,
+                width = rawSize[0].toInt(),
+                height = rawSize[1].toInt()
```
Comment (HlebMaliborski) [inline lib/src/main/java/fi/epicbot/toster/executor/android/AndroidExecutor.kt:111]: Should we check there index? Is it possible to have index exception?

File: lib/src/main/java/fi/epicbot/toster/model/Config.kt [hunk 2]
```diff
@@ -22,6 +22,7 @@ class Config(
     var permissions: Permissions = Permissions(),
     var testTimeoutMillis: Long = 600 * 1000L,
     var deleteAndInstallApk: Boolean = true,
+    var swipeOffset: SwipeOffset = SwipeOffset(offsetPx = 220, offsetFactor = 0.08)
```
Comment (HlebMaliborski) [inline lib/src/main/java/fi/epicbot/toster/model/Config.kt:25]: Why is 220?

**Review by HlebMaliborski — APPROVED** [review]

File: lib/src/main/java/fi/epicbot/toster/model/Config.kt (hunk 2 above)
Comment (alexander-kulikovskii) [reply lib/src/main/java/fi/epicbot/toster/model/Config.kt:25]: just values from my head
verified for instagram :)

File: lib/src/main/java/fi/epicbot/toster/executor/android/AndroidExecutor.kt (hunk 1 above)
Comment (alexander-kulikovskii) [reply lib/src/main/java/fi/epicbot/toster/executor/android/AndroidExecutor.kt:111]: not really, it's standard output for adb command. If it changed, I guess several command can broke down

**Pull request merged**